expo-boomboomapp/src/components/GenderSelector.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useCallback } from "react";
+import { Gender } from "../services/UserService/userServiceI";
+import useEStyles from "../hooks/useEStyles";
+import { View } from "../../components/Themed";
+import { GenderButton } from "./Buttons/GenderButton";
+import { IconName } from "./Icons/IconName";
+
+type GenderSelectorProps = {
+  selectedGender: Gender;
+  setSelectedGender: React.Dispatch<React.SetStateAction<Gender>>;
+};
+
+export default function GenderSelector({
+  selectedGender,
+  setSelectedGender,
+}: GenderSelectorProps) {
+  const isSelectedGender = useCallback(
```

Comment:
No need to use useCallback here, it is indeed a good practice but for long and resource full function, here it can juste be an normal function ;)